ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1092 @@
+with `wildfly-config-url` command line argument which points to the XML descriptor.
+
+* First, https://github.com/openshift/source-to-image#installation[install the s2i].
+* Second, build the quickstart images which will be placed in the docker local registry
+with names `wildfly-quickstarts/client` and `wildfly-quickstarts/server`.
++
+[source,sh,options="nowrap"]
+----
+cd ${PATH_TO_QUICKSTART_DIR}
+
+s2i build --context-dir ejb-txn-remote-call/client \
+  -e MAVEN_OPTS="-Dcom.redhat.xpaas.repo.jbossorg" -e S2I_IMAGE_SOURCE_MOUNTS=extensions \
+  -e JAVA_OPTS_APPEND='-Dwildfly.config.url=$JBOSS_HOME/standalone/configuration/custom-config.xml' \
+  https://github.com/wildfly/quickstart \
+  quay.io/wildfly/wildfly-centos7 wildfly-quickstarts/client
+
+s2i build --context-dir ejb-txn-remote-call/server \
+  -e MAVEN_OPTS="-Dcom.redhat.xpaas.repo.jbossorg" -e S2I_IMAGE_SOURCE_MOUNTS=extensions \
+  https://github.com/wildfly/quickstart \
+  quay.io/wildfly/wildfly-centos7 wildfly-quickstarts/server
+----
+
+[NOTE]
+====
+The WildFly *s2i* code, environmental properties and information about chain builds
+can be found at https://github.com/wildfly/wildfly-s2i.
+====
+
+The result images `wildfly-quickstarts/client` and `wildfly-quickstarts/server` have to be pushed
+to a docker registry. Then they may be used as images deployed to Kubernetes.
+
+[#_running_on_kubernetes_minikube_setup]
+==== Running on Kubernetes: minikube setup
+
+You may configure the minikube to run a local docker registry with command
+[code,sh]
+----
+minikube addons enable registry
+----
+
+Then you can push images to `$(minikube ip):5000`
+and consume them with address `localhost:5000` in the `*-cr.yaml` inside of the minikube.
+
+[source,sh,options="nowrap"]
+----
+docker tag wildfly-quickstarts/client $(minikube ip):5000/wildfly-quickstarts/client
+docker push $(minikube ip):5000/wildfly-quickstarts/client
+docker tag wildfly-quickstarts/server $(minikube ip):5000/wildfly-quickstarts/server
+docker push $(minikube ip):5000/wildfly-quickstarts/server
+----
+
+[NOTE]
+====
+If pushing of the image fails with authentication error
+it may help to enable an insecure registry for address `$(minikube ip):5000`.
+
+Add the address to the list of the insecure registries,
+normally by editing file  `/etc/docker/daemon.json`.
+
+[source,json,options="nowrap"]
+----
+{
+  "insecure-registries": ["<minikube ip address>:5000"]
+}
+----
+
+Reload the docker daemon with `systemctl daemon-reload; systemctl restart docker`.
+====
+
+The other option is to change the docker context
+https://kubernetes.io/docs/setup/learning-environment/minikube/#use-local-images-by-re-using-the-docker-daemon[to use the minikube one].
+
+[#_running_on_kubernetes_run_with_productname_operator]
+==== Running on Kubernetes: run with {productName} Operator
+
+You deploy the {productName} Operator via Kubernetes `Deployment` object.
+The {productName} Operator listens to changes at other Kubernetes objects
```

Comment:
'The {productName} Operator listens to changes at other Kubernetes objects'

Is "at" typical Kubernetes usage?  "changes to Kubernetes objects" is more typical usage, but Kubernetes may have different language practices.